BotTimer.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using System.Threading.Tasks;
+using System.Timers;
+using Discord.WebSocket;
+
+namespace WaterBot
+{
+    static class BotTimer
+    {
+        static SocketTextChannel _channel;
+        static Timer _timer;
+        public static void Start(SocketTextChannel channel)
+        {
+            _channel = channel;
+
+            // Init timer of 3s intervals
+            _timer = new Timer(3000);
```

Comment:
I would create a constructor that would take a parameter of this time. and bind events.